components/AnimatedImage/library/src/desktopMain/kotlin/org/jetbrains/compose/animatedimage/LocalAnimatedImageLoader.kt

```diff
@@ -0,0 +1,21 @@
+package org.jetbrains.compose.animatedimage
+
+import java.io.FileInputStream
+
+internal class LocalAnimatedImageLoader(private val imageUrl: String) : AnimatedImageLoader() {
+    var cachedBytes: ByteArray? = null
+
+    override suspend fun generateByteArray(): ByteArray {
```

Comment:
I omitted it because the method that calls this one already implement `withContent`, but yeah, can be added in case it's called from somewhere else in the future & to get rid of the warning in IDEA :+1: